cmd/rofl/build/build.go

```diff
@@ -147,6 +155,54 @@ var (
 
 			runScript(manifest, buildRofl.ScriptBundlePost)
 
+			// Perform verification when requested.
+			if doVerify {
+				buildEnclaves := make(map[sgx.EnclaveIdentity]struct{})
+				for _, eid := range eids {
+					buildEnclaves[*eid] = struct{}{}
+				}
+
+				manifestEnclaves := make(map[sgx.EnclaveIdentity]struct{})
+				for _, eid := range deployment.Policy.Enclaves {
+					manifestEnclaves[eid] = struct{}{}
+				}
+
+				if !maps.Equal(buildEnclaves, manifestEnclaves) {
+					fmt.Println("Built enclave identities DIFFER from manifest enclave identities!")
```

Comment:
Can you also print out the build enclave ID and the one that is in manifest file?